src/consts.ts

```diff
@@ -26,6 +28,7 @@ export const MaxU256 =
 
 export const MaxSqrtPrice = 65535383934512647000000000000n
 export const MinSqrtPrice = 15258932000000000000n
+export const MAX_BATCHES_QUERIED = 18n
```

Comment:
What is the reason for this batch limit? Is it due to computations or response size?

Reply:
The limit is due computations. Querying 18 batches required 4.98m out of 5m gas limit. Every tickmap with tick spacing greater than one can be retrieved within single query.

The alephium queries has also default http response size limit, that parameter shouldn't cause problems in our implementation.